code/src/pages/products/ProductsPage.tsx

```diff
@@ -1,7 +1,48 @@
+import AppTypography from "@/components/app-typography/AppTypography";
+import AppBox from "@/components/app-box/AppBox";
+import ProductCard from "@/components/product-card/ProductCard";
 import PageWrapper from "@/layouts/page-wrapper/PageWrapper";
+import { useGetProductsQuery } from "@/store/api/productsApi";
+
+import "./ProductsPage.scss";
 
 const ProductsPage = () => {
-  return <PageWrapper>Products</PageWrapper>;
+  const { data: products, isLoading } = useGetProductsQuery();
+
+  //@TODO Create Skeleton for product items
+  if (isLoading) return <AppTypography>Loading...</AppTypography>;
+
+  const productCards = products
+    ? products
+        .slice(0, 4)
+        .map((product) => <ProductCard key={product.id} product={product} />)
+    : null;
+
+  return (
+    <PageWrapper>
+      <AppBox className="spa-products-page">
+        <AppTypography
+          className="spa-products-page__header"
+          translationKey="productsAll.label"
+          component="h1"
+        />
+        <AppBox className="spa-products-page__info">
+          <AppTypography className="spa-products-page__count" component="span">
+            {products?.length || 0}
```

Comment:
move it above the return statement

Reply:
> move it above the return statement inside the variable. Smt like productCount

For me looks nice. Explain why I should to move it above ? It's too long logic condition ?